### ISSUE TITLE
support ipv6 addresses

### ISSUE THREAD

**Issue by ian-kelling** [digint/btrbk#332]
eg: ssh://fe80::ca60:ff:fe31:6b75

**Comment by cogabe** [comment]
Not quite sure but does it support ssh://[fe80::]? With brackets?

**Comment by ian-kelling** [comment]
> Not quite sure but does it support ssh://[fe80::]? With brackets?

No it doesn't. On first glance, it seems like it would be a good way to deal with with the colon in IP:PORT

**Comment by digint** [comment]
Implemented in [support-ipv6 branch](https://github.com/digint/btrbk/tree/support-ipv6). Try something like this:
```
target send-receive ssh://[fe80::ca60:ff:fe31:6b75]/my/path/
```
Should work, unfortunately I can't test it thoroughly as I don't use ipv6 on my servers doing btrbk, please report back if it works for you.

**Comment by digint** [comment]
Thanks, merged to master

**Comment by digint** [comment]
Included in [btrbk-0.30.0](https://github.com/digint/btrbk/releases/tag/v0.30.0)
